core/backend/src/test/annotations/TextAnnotation.test.ts

```diff
@@ -156,173 +156,177 @@ describe("layoutTextBlock", () => {
     expect(tb.lines.every((line) => line.offsetFromDocument.x === 0)).to.be.true;
   });
 
-  it("splits paragraphs into multiple lines if runs exceed the document width", () => {
-    const textBlock = TextBlock.create({ styleName: "" });
-    textBlock.width = 6;
-    textBlock.appendRun(makeTextRun("ab"));
-    expect(doLayout(textBlock).lines.length).to.equal(1);
-    textBlock.appendRun(makeTextRun("cd"));
-    expect(doLayout(textBlock).lines.length).to.equal(1);
-
-    textBlock.appendRun(makeTextRun("ef"));
-    expect(doLayout(textBlock).lines.length).to.equal(2);
-    textBlock.appendRun(makeTextRun("ghi"));
-    expect(doLayout(textBlock).lines.length).to.equal(2);
-
-    textBlock.appendRun(makeTextRun("jklmnop"));
-    expect(doLayout(textBlock).lines.length).to.equal(3);
-
-    textBlock.appendRun(makeTextRun("q"));
-    expect(doLayout(textBlock).lines.length).to.equal(4);
-    textBlock.appendRun(makeTextRun("r"));
-    expect(doLayout(textBlock).lines.length).to.equal(4);
-    textBlock.appendRun(makeTextRun("stu"));
-    expect(doLayout(textBlock).lines.length).to.equal(4);
-
-    textBlock.appendRun(makeTextRun("vwxyz"));
-    expect(doLayout(textBlock).lines.length).to.equal(5);
-  });
-
-  function expectLines(input: string, width: number, expectedLines: string[]): TextBlockLayout {
-    const textBlock = TextBlock.create({ styleName: "" });
-    textBlock.width = width;
-    const run = makeTextRun(input);
-    textBlock.appendRun(run);
+  // Node in the mobile add-on does not include Intl, so these tests fail. Right now, mobile users
+  // are not expected to do any editing, but long term we will attempt to find a better solution.
+  if (!ProcessDetector.isMobileAppFrontend) {
```

Comment:
Looks like you brought along a bunch of formatting changes as well, maybe revert those so we keep our author commit history/blame in tact?

Reply:
Should be able to do something like the following inside a test:
```
  if (ProcessDetector.isMobileAppFrontend) {
    this.skip();
  }
```

Reply:
@pmconne, that's not working here, and I have no idea why. For some reason, `this` may be undefined. If I uses `this?.skip()` I get a different error: Property 'skip does not exist on type 'never'.

Reply:
All the other places that call this.skip() use async function () instead of () =>. I guess I could try that.